src/main/java/nl/dtls/fairdatapoint/service/impl/FairMetaDataMetricsServiceImpl.java

```diff
@@ -55,7 +55,7 @@
 @Component
 public class FairMetaDataMetricsServiceImpl implements FairMetaDataMetricsService {
 
-    private static final Logger LOGGER = LogManager.getLogger(FairMetaDataServiceImpl.class
+    private static final Logger LOGGER = LoggerFactory.getLogger(FairMetaDataServiceImpl.class
             .getName());
```

Comment:
We can use the class itself instead of resolving it per name. (also for consistency)